packages/workspace/src/core/project-graph/require-resolve.ts

```diff
@@ -0,0 +1,7 @@
+export function requireResolve(p: string) {
+  try {
+    return require.resolve(p);
```

Comment:
This should behave like the following:

```
@angular/core -> node_modules/@angular/core
react -> node_modules/react
```

I think it could be something like:

1. Resolve the `package.json` of the package.
2. Get the directory of the `package.json`
3. Get that relative from the app root

```ts
const appRoot = import from app root
...
function resolveNpmPackageRoot(p: string) {
  try {
    return relative(appRoot, dirname(require.resolve(join(p, 'package.json))));
  } catch (e) { return '' }
}
```

Reply:
Would you not want the absolute path on disk though? That way there isn't a need to figure out relative paths later downstream.

Reply:
It should be synonymous with what users would write in `workspace.json`:

`apps/proj1`